code/modules/projectiles/projectile/bullettypes.dm

```diff
@@ -201,6 +232,16 @@ There are important things regarding this file:
 	embed = FALSE
 	sharp = FALSE
 
+/obj/item/projectile/bullet/shotgun/beanbag/s
+	damage_types = list(BRUTE = 9)
+	agony = 55
+
+/obj/item/projectile/bullet/shotgun/beanbag/bd
```

Comment:
No.

Reply:
This has been approved by Reere 2.0, aka Firefox on the condition it's admin only.